src/main/java/org/itxtech/synapseapi/SynapsePlayer.java

```diff
@@ -73,7 +74,7 @@ public SynapseEntry getSynapseEntry() {
     }
 
     @Override
-    protected void processLogin() {
+    public void processLogin() {
```

Comment:
Why?

Reply:
just for my testing, you can make protected back